### PR TITLE
Fix abs

### PR DESCRIPTION
**Pull request by Rubtsowa**

### CODE REVIEW

**Comment by Alexander-Makaryev** [comment]
@Rubtsowa 
Could you please look into https://github.com/IntelPython/dpnp/pull/127?

File: dpnp/backend/custom_kernels_mathematical.cpp [hunk 1]
```diff
@@ -35,7 +35,7 @@ template <typename _KernelNameSpecialization>
 class custom_elemwise_absolute_c_kernel;
 
 template <typename _DataType>
-void custom_elemwise_absolute_c(void* array1_in, const std::vector<long>& input_shape, void* result1, size_t size)
+void custom_elemwise_absolute_c(void* array1_in, void* result1, size_t size)
```
Comment (shssf) [inline dpnp/backend/custom_kernels_mathematical.cpp:38]: Why parameter `input_shape` is removed? What do we expect from the function (we do not need axis specific algorithm)?

Comment (Alexander-Makaryev) [reply dpnp/backend/custom_kernels_mathematical.cpp:38]: @shssf There is no axis param in function absolute.
https://numpy.org/doc/stable/reference/generated/numpy.absolute.html

Comment (Rubtsowa) [reply dpnp/backend/custom_kernels_mathematical.cpp:38]: @shssf  absolute has not atribute `axis`.
https://numpy.org/doc/stable/reference/generated/numpy.absolute.html

**Comment by shssf** [comment]
is this PR ready for merge?

**Comment by Alexander-Makaryev** [comment]
@Rubtsowa 
cdef size_t input_shape_size = input.ndim
https://github.com/IntelPython/dpnp/blob/master/dpnp/backend_mathematical.pyx#L72
Is still not used anywhere.
Could you please check one more time my comments in your original PR?
https://github.com/IntelPython/dpnp/pull/104

**Pull request merged**